docs/getting-started.md

```diff
@@ -107,16 +107,67 @@ you've already synced with the Radicle network.
 - `rad ls`: View a list of your synced Radicle projects.
 - `rad [command] --help`: See additional options for an individual command.
 
+Read more about CLI workflows in the [`rad` man
+page](https://github.com/radicle-dev/radicle-cli/blob/master/rad.1.adoc).
+
+## Install Radicle Upstream
+
+Now that your project has been successfully pushed to the Radicle network, you can also use the
+[Upstream](https://radicle.xyz) desktop application to view and manage your project.
+
+The first step is to [download and install the Upstream client](https://radicle.xyz/tryit) for your operating system.
+
+> ⚠️ Please note: We currently only support MacOS and Linux. Windows is coming soon!
+
+On initial launch, the Upstream client copies the `radicle-cli` tool to your `$HOME` directory then runs a few system
```

Comment:
It actually copies the `upstream` cli binary and the `git-remote-rad` helper to `$HOME/.radicle/bin`, not the `radicle-cli` tool.